cpp/zeder_to_zotero_importer.cc

```diff
@@ -48,7 +48,8 @@ using namespace ZoteroHarvester;
             "\t                                Special-case for updating: Use '*' to update all entries found in the config that belong to the Zeder flavour\n"
```

Comment:
I think you should add "or *" here.

Reply:
I don't understand... it already says "Use '*'"?